### ISSUE TITLE
Getting fatal error when creating host.

### ISSUE THREAD

**Issue by autocoder10** [ansible-collections/ibm.spectrum_virtualize#31]
##### SUMMARY
When running a create host playbook using ibm_svc_host py module, got a fatal error and showing as Failed but creating the host entry.

**Error:**
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Module failed. Error [argument of type 'NoneType' is not iterable]."}

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
ibm_svc_host

##### ANSIBLE VERSION

```paste below
ansible 2.9.17
```

##### CONFIGURATION

```paste below

```

##### STEPS TO REPRODUCE

```yaml
---
- name: Using the IBM Spectrum Virtualize collection
  collections:
    - ibm.spectrum_virtualize
  gather_facts: no
  connection: local
  hosts: localhost

  vars:
    clustername: testcluster
    username: ansibletest
    password: passw0rd

  tasks:
    - name: Create host entry
      ibm_svc_host:
        clustername: "{{clustername}}"
        domain:
        username: "{{ username }}"
        password: "{{ password }}"
        log_path: /tmp/svcplaybook.debug
        validate_certs: false
        name: test1
        state: present
        fcwwpn: 11223344556633AA:11223344556633BB
        iogrp: 0
        protocol: scsi
        type: generic
```

##### ACTUAL RESULTS

```paste below
2021-03-08T14:35:39.209  INFO 140378744436544 ibm_svc_host.py:apply():372 CHANGED: host does not exist, but requested state is 'present'
2021-03-08T14:35:39.209  INFO 140378744436544 ibm_svc_host.py:host_create():272 creating host 'test1'
2021-03-08T14:35:39.209  INFO 140378744436544 ibm_svc_host.py:host_create():290 creating host command '11223344556633AA:11223344556633BB' opts 'generic'
```

**Comment by autocoder10** [comment]
I did some troubleshooting and after the SVC code updated from 8.2.1.8 to 8.3.1.3 I believe this issue started. Can you please check and let me know that fatal error is occurring.

**Comment by rohitk-github** [comment]
Thanks for reporting it @autocoder10 
We will analyze this and revert.

**Comment by rohitk-github** [comment]
Hi @autocoder10 Yes, I have an update on this. This issue is present in SV 8.3.1.3 and 8.3.1.4. Apparently, the REST calls for "mk" actions do not return JSON output and thus "json.load()" is failing. 
This is fixed in Build 8.4.0.0.

**Comment by autocoder10** [comment]
Hmm that's a bummer. We are not planning to upgrade SVC's to 8.4 this year. So, will have to add some addl. validation tasks in the playbook after creating the objects before going to the next task till then.
